controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -591,7 +592,7 @@ bool master_thread::autoconfig_wsc_add_m2(std::shared_ptr<ieee1905_1::tlvWscM1>
 bool master_thread::handle_cmdu_1905_autoconfiguration_WSC(Socket *sd,
                                                            ieee1905_1::CmduMessageRx &cmdu_rx)
 {
-    // Check if this is a M2 message that we sent to the agent, which was just looped back.
+    // Check if this is a M2 message that we sent to the agent, which was just looped back,
```

Comment:
Why this change?

The capitalization of the following line should be changed as well then.

Reply:
mistake, will revert